### PR TITLE
Add errno label to fuse ops metric

### PR DESCRIPTION
**Pull request by polyrabbit**
Senario:
1. Better understanding users' pattern
2. Set up alarm rules on abnormal error rate
image

### CODE REVIEW

File: pkg/vfs/accesslog.go [hunk 1]
```diff
@@ -30,7 +32,7 @@ var (
 		Name:    "fuse_ops_durations_histogram_seconds",
 		Help:    "Operations latency distributions.",
 		Buckets: prometheus.ExponentialBuckets(0.0001, 1.5, 30),
-	}, []string{"method"})
+	}, []string{"method", "errno"})
```
Comment (davies) [inline pkg/vfs/accesslog.go:35]: there will be too many metrics, especially for hisogram

Comment (davies) [reply pkg/vfs/accesslog.go:35]: Maybe we can have a single metric for all errors.

Comment (polyrabbit) [reply pkg/vfs/accesslog.go:35]: By `too many metrics`, do you mean too many metrics on the client side or tsdb side?

What do you think I move the statistic to a counter, such as:
```suggestion
	opsIOErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
		Name: "fuse_ops_io_errors",
		Help: "Number of IO errors.",
	}, []string{"method", "errno"})
```

Comment (davies) [reply pkg/vfs/accesslog.go:35]: both.

The combination of method and errno could be hundreds. A single counter or with single label should be enough for monitoring. For further investing, we should collect the accesslog to find out the details.

Comment (polyrabbit) [reply pkg/vfs/accesslog.go:39]: Updated - added a new metric for io errors

Comment (davies) [reply pkg/vfs/accesslog.go:39]: The new metric has too much labels, please pick one

Comment (polyrabbit) [reply pkg/vfs/accesslog.go]: Done

File: pkg/vfs/accesslog.go [hunk 2]
```diff
@@ -31,6 +33,10 @@ var (
 		Help:    "Operations latency distributions.",
 		Buckets: prometheus.ExponentialBuckets(0.0001, 1.5, 30),
 	}, []string{"method"})
+	opsIOErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "fuse_ops_io_errors",
+		Help: "Number of IO errors.",
+	}, []string{"errno"})
```
Comment (davies) [inline pkg/vfs/accesslog.go:39]: We should add the `errno` into the name of metric in `.stats`, see internal.go

Comment (polyrabbit) [reply pkg/vfs/accesslog.go:39]: Do you mean change `juicefs_fuse_ops_io_errors` to `juicefs_fuse_ops_io_errors_errno` in .stats file?

Comment (davies) [reply pkg/vfs/accesslog.go:39]: yes

Comment (polyrabbit) [reply pkg/vfs/accesslog.go:47]: Added

**Pull request merged**